src/pages/profile/Profile.jsx

```diff
@@ -37,175 +51,153 @@ const Profile = () => {
   const [profileLastName, setProfileLastName] = useState("");
   const [profileUsername, setProfileUsername] = useState("");
   const [profileProfilePicture, setProfileProfilePicture] = useState(null);
-  const [collections, setCollections] = useState([]);
-  const [pages, setPages] = useState(-1);
-  const [hasMore, setHasMore] = useState(true);
+  const [mode, setMode] = useState(COLLECTIONS_MODE);
+  const [loading, setLoading] = useState(false);
 
-  // TODO: add api call for username
-  const getUserInformation = useCallback(() => {
-    if (currentUserId) {
-      getCurrentUser()
-        .then((res) => {
-          setProfileUserId(Number(currentUserId));
-          setProfileFirstName(res.firstName);
-          setProfileLastName(res.lastName);
-          setProfileUsername(res.username);
-          setProfileProfilePicture(res.pictureUrl);
-        })
-        .catch((e) => console.log(e));
-    } else {
-      if (username) {
-        getUserByUsername()
-          .then((res) => {
-            setProfileUserId(Number(res.userId));
-            setProfileFirstName(res.firstName);
-            setProfileLastName(res.lastName);
-            setProfileUsername(res.username);
-            setProfileProfilePicture(res.pictureUrl);
-          })
-          .catch((e) => console.log(e));
-      }
-    }
-  }, [currentUserId, username]);
+  const isOwnProfile = parseInt(currentUserId) === profileUserId;
```

Comment:
I'm not sure if this variable being a constant will cause any weird effects. E.g. if I go navigate between profiles. Do we need to make this a state?

Reply:
const values should automatically be recomputed whenever profileUserId changes